workflow/scripts/modify_prenetwork.py

```diff
@@ -332,6 +332,14 @@ def transmission_costs_from_modified_cost_data(n, costs, transmission, length_fa
     )
     n.links.loc[dc_b, "capital_cost"] = costs
 
+def must_run_biomass(n, p_min_pu, regions):
+    """
+    Set p_min_pu for biomass generators to the specified value.
+    """
+    logger.info(f"Setting p_min_pu = {p_min_pu} for biomass generators")
+    links_i = n.links[(n.links.carrier == 'urban central solid biomass CHP') & (n.links.bus0.str.startswith(tuple(regions)))].index
```

Comment:
at the moment, the must-run condition us on the CHPs which leads to too high share of heat|biomass.
instead the must-run condition should be on the electricity only `solid biomass` plants.

in 2020 there are 7.81 GW of electricity only solid biomass plants.